spec/lib/tasks/push_aliquot_data_to_warehouse.rake_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rake'
+
+# only load Rake tasks if they haven't been loaded already
+Rails.application.load_tasks if Rake::Task.tasks.empty?
+
+RSpec.describe 'RakeTasks' do
+  # Create a default SMRTLink version to be used by factories
+  before do
```

Comment:
We have to initialise this smrtlink default all over the place, should definitely move it to a test helper that sets it up on load, probably not in scope for this story.

Reply:
Yes, it is inconvenient to do it in every single place a run is created. Probably make sense to add as a separate story, I hope.